Assets/MixedRealityToolkit.Tests/PlayModeTests/TestHand.cs

```diff
@@ -50,30 +50,60 @@ public IEnumerator Hide()
             yield return new WaitForFixedUpdate();
         }
 
-        public IEnumerator MoveTo(Vector3 newPosition, int numSteps = 30)
+        public IEnumerator MoveToSteps(Vector3 newPosition, int numSteps)
```

Comment:
Can we have a single MoveTo method defaulting numSteps to `PlayModeTestUtilities.HandMoveStepsDefault`?

Reply:
I was thinking that having an separate name for the explicit step version would be more clear, but no strong opinion on that, i will change it.

_Minor technical note_: i can't use the `PlayModeTestUtilities.HandMoveStepsDefault` directly as a default for the parameter, because it's not a runtime const. Instead i would just use two overloads for the same function, which amounts to the same thing.